automated install/basic-install.sh

```diff
@@ -188,7 +188,7 @@ os_check() {
         local remote_os_domain valid_os valid_version valid_response detected_os detected_version display_warning cmdResult digReturnCode response
         remote_os_domain="versions.pi-hole.net"
 
-        detected_os=$(grep "\bID\b" /etc/os-release | cut -d '=' -f2 | tr -d '"')
+        detected_os=$(grep "\\bID\\b" /etc/os-release | cut -d '=' -f2 | tr -d '"')
```

Comment:
An alternative would be to use single quotes here. Or better be more explicit:
```
grep '^ID=' /etc/os-release
```

Reply:
Thanks for the feedback :)

Updated it.